com.unity.perception/Runtime/GroundTruth/Labelers/Visualization/ControlPanel.cs

```diff
@@ -0,0 +1,128 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.UI;
+
+namespace UnityEngine.Perception.GroundTruth
+{
+    /// <summary>
+    /// The control panel for labeler visualizers. This panel resides in the upper right hand corner of
+    /// the display. Any UI element can be added to the panel, but it contains helper methods to quickly
+    /// create some of the most common control panel display elements: toggles and sliders.
+    /// </summary>
+    public class ControlPanel : MonoBehaviour
```

Comment:
For completeness, let there be a way of reading the list of controls. Also, what happens when two different labelers want to use the same name for their control (like "enabled")? Unlikely, but consider a model where we don't have keys and instead just add and remove GameObjects

Reply:
Ok, moved away from labels and added a list Getter

Reply:
OK, updated with your suggestions.

Reply:
`ToList` allocates a new list and copies the objects, which is not desirable. [This page has some relevant bits](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/guidelines-for-collections).

I'd suggest changing it to `IEnumerable<GameObject>`

Reply:
Makes sense, updated.